config/webpack/webpack.common.js

```diff
@@ -185,7 +186,11 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
             },
             {
                 test: /\.css$/i,
-                use: ['style-loader', 'css-loader'],
+                use: ['style-loader', 'file-loader', 'css-loader'],
+            },
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/i,
+                type: 'asset/resource',
```

Comment:
You can just use `asset` instead of `asset/resource`, and that will automatically optimize whether the asset should be inline or a separate file, depending on it's size:

```suggestion
                type: 'asset',
```